settings/DevHome.Settings/ViewModels/ExtensionSettingsViewModel.cs

```diff
@@ -0,0 +1,86 @@
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using AdaptiveCards.Rendering.WinUI3;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.WinUI;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Common.Views;
+using DevHome.Logging;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.ApplicationModel;
+
+namespace DevHome.Settings.ViewModels;
+
+public partial class ExtensionSettingsViewModel : ObservableObject
+{
+    public ExtensionSettingsViewModel()
+    {
+        Breadcrumbs = new ObservableCollection<Breadcrumb> { };
+    }
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs
+    {
+        get; set;
+    }
+
+    [RelayCommand]
+    private async Task OnSettingsContentLoadedAsync(PluginAdaptiveCardPanel pluginAdaptiveCardPanel)
+    {
+        var pluginWrappers = Task.Run(async () =>
+        {
+            var pluginService = Application.Current.GetService<IPluginService>();
+            return await pluginService.GetInstalledPluginsAsync(true);
+        }).Result;
+
+        var navigationService = Application.Current.GetService<INavigationService>();
+        foreach (var pluginWrapper in pluginWrappers)
+        {
+            if ((navigationService.LastParameterUsed != null) &&
+                ((string)navigationService.LastParameterUsed == pluginWrapper.PackageFullName))
+            {
+                var stringResource = new StringResource("DevHome.Settings/Resources");
+                Breadcrumbs.Add(new Breadcrumb(stringResource.GetLocalized("Settings_Header"), typeof(SettingsViewModel).FullName!));
+                Breadcrumbs.Add(new Breadcrumb(stringResource.GetLocalized("Settings_Extensions_Header"), typeof(ExtensionsViewModel).FullName!));
+                Breadcrumbs.Add(new Breadcrumb(pluginWrapper.Name, typeof(ExtensionSettingsViewModel).FullName!));
+
+                var settingsProvider = Task.Run(() => pluginWrapper.GetProviderAsync<ISettingsProvider>()).Result;
+                if (settingsProvider != null)
+                {
+                    var adaptiveCardSessionResult = settingsProvider.GetSettingsAdaptiveCardSession();
+                    if (adaptiveCardSessionResult.Result.Status == ProviderOperationStatus.Failure)
+                    {
+                        GlobalLog.Logger?.ReportError($"{adaptiveCardSessionResult.Result.DisplayMessage} - {adaptiveCardSessionResult.Result.DiagnosticText}");
+                        await Task.CompletedTask;
+                    }
+
+                    var adpativeCardSession = adaptiveCardSessionResult.AdaptiveCardSession;
+                    var renderer = new AdaptiveCardRenderer();
+                    renderer.HostConfig.ContainerStyles.Default.BackgroundColor = Microsoft.UI.Colors.Transparent;

```

Comment:
Nit: You probably want to set the host config to the file that we use (should be an example in Settings, there's one in Dashboard but we use a slightly different hostconfig file there). And then overwrite the specific BackgroundColor property.
It's ok if you don't, just adaptive cards will look less DevHome-y until you do.

Reply:
Apparently, it's the opposite.  Not transparent is the default.  When I removed this and Dev Home was in dark theme, the adaptive card just looked like a big white box :)